components/rt/src/rt_estimator.m

```diff
@@ -214,6 +214,6 @@
 Tdate = cell2table(longdates(18+(1:size(M,2)))','VariableNames',{'Date'});
 TTout = [Tdate,TTout];
 filename = [today '_Rt_estimate.csv'];
-writetable(TTout,[outFolder filesep filename])
+writetable(TTout,["output/" filename])
```

Comment:
`filesep` is meant to add the `/` across platforms. On Windows, `filesep` corresponds to `\`. Can you revert?